omnibus/config/projects/chef-workstation.rb

```diff
@@ -78,11 +83,10 @@
   dependency "chef-dk-powershell-scripts"
 end
 
+
 dependency "version-manifest"
 dependency "openssl-customization"
 
-dependency "stunnel" if fips_mode?
```

Comment:
Hmm - seems like not supporting a fips distribution of Chef Workstation is going to break some of our customers. I realize we don't have the pipeline right now but this seems like an issue we should address.

Reply:
The path we've followed in other products is to make every build fips-capable, and control the behavior via feature flag.  If we determine we need FIPs support in DK, I'm hoping we're able to do it the same way.

Reply:
Got clarification from @robbkidd - this will be true for any fips-supported platform (rhel, windows) and not based on build configuration. I'll keep it in.